### PR TITLE
[MBL-16182][Student][Teacher] Small UI improvements

### PR DESCRIPTION
**Pull request by kristofnemere**
refs: MBL-16182
affects: Student, Teacher
release note: UI improvements.

test plan:
- test theme selector bottom sheet on tablet/phone landscape
- test page edit screen, it is no longer a dialog
- test teacher settings in master detail (should open detail fragments correctly to the right)

## Checklist

- [x] Follow-up e2e test ticket created or not needed
- [x] A11y checked
- [x] Approve from product or not needed

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

File: apps/student/src/main/java/com/instructure/student/util/StudentToolbarSetup.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+package com.instructure.student.util
+
+import android.app.Activity
+import androidx.appcompat.widget.Toolbar
+import com.instructure.pandautils.utils.ToolbarSetup
+import com.instructure.pandautils.utils.ViewStyler
+import com.instructure.pandautils.utils.setupAsBackButton
+
+class StudentToolbarSetup : ToolbarSetup {
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/student/src/main/java/com/instructure/student/util/StudentToolbarSetup.kt]: Thanks, fixed it.

File: apps/teacher/src/main/java/com/instructure/teacher/di/ToolbarSetupModule.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.FragmentComponent
+
+@Module
+@InstallIn(FragmentComponent::class)
+class ToolbarSetupModule {
+    @Provides
+    fun provideToolbarSetup(): ToolbarSetup {
```
[the remark this answers is not in the document]
Comment (kristofnemere) [reply apps/teacher/src/main/java/com/instructure/teacher/di/ToolbarSetupModule.kt]: Thanks, fixed it.

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Pull request merged**